박수인/conter/src/App.jsx

```diff
@@ -0,0 +1,16 @@
+import "./App.css";
+import Counter from "./CounterPage";
+
+function App() {
+  return (
+    <>
+      <div>
+        <div>
+          <Counter />
+        </div>
+      </div>
```

Comment:
div 태그로 여러번 감싸신 이유가 있을까요?
의미없는 태그를 중첩해서 사용하면, 가독성이 떨어질 뿐만 아니라 성능상으로 문제가 야기될 수 있습니다! 컨테이너로 사용하실 용도면 하나의 div 태그로 감싸시는건 어떠신가요?
```suggestion
      <div>
          <Counter />
      </div>
```

Reply:
앗 그러네요 작성시에 잘못 들어간 것 같습니다 알려주신 내용 잘 참고하겠습니다!